scalafix-core/shared/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -108,23 +109,18 @@ object EscapeHatch {
     .warning("", "Unused Scalafix suppression. This can be removed")
     .withOwner(RuleName("UnusedScalafixSuppression"))
 
+  private type EscapeOffset = Int
```

Comment:
Out of curiosity, is there a motivation besides saving allocations to make this a type alias? In general, I'm in favor of wrapper classes since they provide compile-time guarantees but I don't mind it much.

Reply:
I thought it was a bit overkill because when we need to look up something in the `TreeMap`, first we need to wrap `Position.start|end` (which are `Int`s) into the case class, which adds some boilerplate:

```scala
escapeTree.to(EscapeOffset(position))
```

There is only one exception to that. Besides, `EscapeOffset` is private.

It is not a big deal though... I actually favor wrapper/value classes too, but didn't see much benefit for this particular case. I can revert the change if you prefer 😄

Reply:
@marcelocenerine I would prefer to revert the change. Making EscapeOffset a value class helped me a lot to solve the problem.

Reply:
ok, I will revert it